store/src/java/com/zimbra/cs/service/mail/GetContacts.java

```diff
@@ -203,7 +210,14 @@ static void partitionItems(ZimbraSoapContext lc, ArrayList<ItemId> ids, ArrayLis
     throws ServiceException {
         List<Element> responses = new ArrayList<Element>();
 
-        // note that we removed all <cn> elements from the request during handle(), above
+        //remove all 'contact' elements from original request
+        for (Element e : request.listElements()) {
```

Comment:
There is a `List<Element> listElements(String name)` method too which may be worth using?